kfdef/kfctl_openshift_ml-pipelines.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
+metadata:
+  name: pipelines
+  namespace: odh-applications
+spec:
+  applications:
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: odh-common
+      name: odh-common
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: jupyterhub/jupyterhub
+      name: jupyterhub
+    - kustomizeConfig:
+        overlays:
+          - additional
+        repoRef:
+          name: manifests
+          path: jupyterhub/notebook-images
+      name: notebook-images
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: odh-dashboard
+      name: odh-dashboard
```

Comment:
Since odh-manifests is using a pinned version of odh-dashboard and this kfdef is relying on in-development features for pipelines and the dasbhboard/pipelines UI then it would probably be better to include the odh-dashboard manifest from the odh-dashboard repo. 

See the odh-dasbhboard [kfdef](https://github.com/opendatahub-io/odh-dashboard/blob/main/manifests/kfdef/odh-dashboard-kfnbc-test.yaml) for an example on the `kustomizeConfig` and additional `manifests-dashboard` `repos` item